flexbox/src/main/res/layout/activity_minheight_test.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please put this file under flexbox/src/androidTest/res
and put license notice here